### PR TITLE
[#9182] Release V6.13.0

### LINKED ISSUE #9182: Release V6.13.0
Scheduled for Saturday, Oct 13, 2018, 10.00am SGT.

Release Lead:

- [x] Tagged the latest stable release code with the latest version number
- [x] Released changelogs

PM:

- [ ] Deployed the new version
- [ ] Released the new version for end users

### PR DESCRIPTION
**Pull request by xpdavid**
Part of #9182

### CODE REVIEW

**Comment by xpdavid** [comment]
Updated aboutPage: https://6-13-0-dot-teammates-xiaopu.appspot.com/about.jsp

File: src/main/webapp/data/developers.json [hunk 1]
```diff
@@ -1601,21 +1613,41 @@
       "username": "darrenwee",
       "startPeriod": "Jan 2018"
     },
+    {
+      "name": "Jacob Li",
+      "username": "jacoblipech",
+      "image": "images/teammembers/jacob.png",
+      "startPeriod": "Aug 2018"
```
Comment (wkurniawan07) [inline src/main/webapp/data/developers.json]: They started at May 18, no?

File: src/main/webapp/data/developers.json [hunk 2]
```diff
@@ -1601,21 +1613,41 @@
       "username": "darrenwee",
       "startPeriod": "Jan 2018"
     },
+    {
+      "name": "Jacob Li",
```
Comment (wkurniawan07) [inline src/main/webapp/data/developers.json]: You should remove entries about them under contributors.

File: src/main/webapp/data/developers.json (hunk 1 above)
Comment (xpdavid) [reply src/main/webapp/data/developers.json]: Should it be the time they get promoted?

Comment (wkurniawan07) [reply src/main/webapp/data/developers.json]: If you look around, it's not the case for everyone else (you and me included).

File: src/main/webapp/data/developers.json (hunk 2 above)
Comment (xpdavid) [reply src/main/webapp/data/developers.json]: @wkurniawan07 Changes made :)

**Pull request merged**